qimage_conversions.cpp: Fix build on big endian systems

Backporting an upstream patch to fix a qt5base build failure on big
endian systems. This is the error message:

image/qimage_conversions.cpp:2257:9: error: expected '}' before numeric 
constant

Upstream commit:

  https://qt.gitorious.org/qt/qtbase/commit/404f4281fda764cafdaa5635db995dabc4f1de8c

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>

From 404f4281fda764cafdaa5635db995dabc4f1de8c Mon Sep 17 00:00:00 2001
From: Dmitry Shachnev <mitya57@gmail.com>
Date: Tue, 9 Dec 2014 09:27:53 +0300
Subject: [PATCH] qimage_conversions.cpp: Fix build on big endian systems

Change-Id: I8149eb2deaa101daf85a957ff48c3a7140c43bbc
Reviewed-by: Timo Jyrinki <timo.jyrinki@canonical.com>
Reviewed-by: Allan Sandfeld Jensen <allan.jensen@theqtcompany.com>
---
 src/gui/image/qimage_conversions.cpp |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gui/image/qimage_conversions.cpp b/src/gui/image/qimage_conversions.cpp
index 17563b1..858a0d9 100644
--- a/src/gui/image/qimage_conversions.cpp
+++ b/src/gui/image/qimage_conversions.cpp
@@ -2252,7 +2252,7 @@ Image_Converter qimage_converter_map[QImage::NImageFormats][QImage::NImageFormat
         convert_ARGB_to_ARGB_PM,
 #else
         0,
-        0
+        0,
 #endif
         0, 0, 0, 0
     }, // Format_RGBA8888
@@ -2281,7 +2281,7 @@ Image_Converter qimage_converter_map[QImage::NImageFormats][QImage::NImageFormat
 #else
         0,
         0,
-        0
+        0,
 #endif
         0, 0, 0, 0
     }, // Format_RGBA8888_Premultiplied
-- 
1.7.1

